actions/packages.py

```diff
@@ -346,10 +352,10 @@ def runTransaction(transaction_data, cache_only=None):
     zypper = Zypper()
     return __strip_message(zypper.transact(transaction_data))
 
-def fullUpdate(force=0, cache_only=None):
+def fullUpdate(force=0, cache_only=None, allow_vendor_change=False):
     """ Update all packages on the system. """
     zypper = Zypper(download_only=cache_only)
-    return __strip_message(zypper.update())
+    return __strip_message(zypper.update(allow_vendor_change))
```

Comment:
```suggestion
    return __strip_message(zypper.update(allow_vendor_change=allow_vendor_change))
```
zypper.update() expects also a package list. So you need to name the parameter you want to set.